tasks/configure.yml

```diff
@@ -44,6 +44,25 @@
   tags:
     - rstudio-workbench-configure-load-balancer
 
+- name: Create a directory for workbench in EFS
+  file:
+    path: "{{ rstudio_workbench_nfs_mount_path }}/workbench"
+    state: directory
+    mode: 0755
+  when:
+    - rstudio_workbench_nfs_mount_path is defined
+
+- name: Create a sub-directories used by workbench
+  file:
+    path: "{{ rstudio_workbench_nfs_mount_path }}/workbench/{{ item }}"
```

Comment:
Whole EFS is dedicated to Workbench, do we need to specify `workbench` here explicitly?